data-service/src/main/java/uk/gov/gchq/palisade/service/data/request/AddCacheRequest.java

```diff
@@ -15,7 +15,7 @@
  */
 package uk.gov.gchq.palisade.service.data.request;
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
```

Comment:
There is a dependency conflict here somewhere! Some classes are using different versions of Jackson. Check the package names! com.fasterxml is the up to date Jackson! Please check the dependency list that this module is pulling in.